docker/manager.py

```diff
@@ -0,0 +1,821 @@
+#!/usr/bin/env python3
+import errno
+import inspect
+import json
+import logging
+import os
+import pathlib
+import re
+import shutil
+from collections import defaultdict
+from collections.abc import Iterable
+from copy import deepcopy
+from typing import Dict, Optional
+
+from absl import flags, app
+from cerberus import Validator
+from glom import glom, Path, PathAccessError, Assign, PathAssignError
+from jinja2 import Environment
+from ruamel import yaml
+from prettytable import PrettyTable
+
+logging.basicConfig(level=logging.NOTSET)
```

Comment:
There is currently a bug somewhere that prevent `logging` to setup basic config. Could be `cerberus` related.